packages/dds/merge-tree/src/client.ts

```diff
@@ -566,7 +566,7 @@ export class Client {
                 throw new LoggingError(
                     "RangeOutOfBounds",
                     {
-                        UsageError: true,
```

Comment:
can use the UsageError  class here?

Reply:
I wasn't sure if DDSes should be using Container-Utils' error classes.  Is Container-Utils meant to hold anything general purpose to code running in a container or something?

Reply:
i think it's already in the dependency tree, but yeah, probably shouldn't be. no need to fix this. rather get to common props